src/software/pipeline/main_cameraInit.cpp

```diff
@@ -363,13 +363,21 @@ int aliceVision_main(int argc, char **argv)
 
   // check sensor database
   std::vector<sensorDB::Datasheet> sensorDatabase;
-  if(!sensorDatabasePath.empty())
+  if (sensorDatabasePath.empty())
```

Comment:
This is a change in behavior of the program.
Until now we can run the program without any sensor database (because e.g. we provide K).
With these changes, we are now forced to have a sensor database.
I would keep it like it was, it's just a matter of replacing the throwing with a LOG_INFO/WARNING and still checking whether `sensorDatabasePath` is empty.